src/cmd/support_archive/objects_query.go

```diff
@@ -0,0 +1,81 @@
+package support_archive
+
+import (
+	"github.com/Dynatrace/dynatrace-operator/src/webhook"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type objectQuery struct {
+	groupVersionKind schema.GroupVersionKind
+	listOptions      []client.ListOption
+}
+
+func getObjectQueries(namespace string) []objectQuery {
+	return []objectQuery{
+		{
+			groupVersionKind: schema.GroupVersionKind{
+				Group:   "",
+				Version: "v1",
+				Kind:    "NamespaceList",
+			},
+			listOptions: []client.ListOption{
+				client.HasLabels{
+					webhook.InjectionInstanceLabel,
+				},
+			},
+		},
+		{
+			groupVersionKind: schema.GroupVersionKind{
+				Group:   "",
+				Version: "v1",
+				Kind:    "NamespaceList",
+			},
+			listOptions: []client.ListOption{
+				client.MatchingFields{
+					"metadata.name": namespace,
```

Comment:
This file contains the description of how to query the objects we want. It is done this way to keep the query mechanism general and not implemented separately for each object type. I don't want special handling just because the list returns only one object.